docs/setup/aks/setup-aks-appinsights-grafana.md

```diff
@@ -191,17 +191,19 @@ kubectl --namespace default port-forward <REPLACE-WITH-GRAFANA-POD-NAME> 3000
 ```
 Now, browse to http://127.0.0.1:3000/ and log in using the admin username and password you provided in the `values.yaml` file.
 
+
```

Comment:
why the extra new line?

Reply:
leftover from my attempts to get the file to download. Has no effect on publishing but deleted it anyway